echelon/inc/functions.php

```diff
@@ -517,9 +517,15 @@ function tooltip($msg, $float = false) {
  * Echo out simple clientdetails link
  */
 function clientLink($name, $id, $game_id = NULL) {
-	if(!empty($game_id))
+    if(empty($game_id)){

```

Comment:
What we do here is to first check if `$game_id` was passed to the function or not.
If not, we use `global $game` to fill the `$game_id`